src/shared-components/Title1/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Typography } from '@mui/material';
+
+interface Title1Props {
+  children: string | JSX.Element;
+}
+
+export const Title1 = ({ children }: Title1Props) => {
+  return (
+    <Typography variant='h2' sx={{ textAlign: 'center', margin: '10px' }}>
```

Comment:
It would be better to use spacing from the theme here (and similar places): `theme.spacing(2);` instead of 10px so that the spacing is consistent across the application